src/data/equiv/transfer_instance.lean

```diff
@@ -141,11 +149,30 @@ protected def mul_one_class [mul_one_class β] : mul_one_class α :=
 let one := e.has_one, mul := e.has_mul in
 by resetI; apply e.injective.mul_one_class _; intros; exact e.apply_symm_apply _
 
+/-- Transfer `mul_zero_class` across an `equiv` -/
```

Comment:
I think this hunk can be removed, and is also a merge conflict

Reply:
It's toast.